test/logic/utility/model/account.sql

```diff
@@ -121,3 +121,34 @@ $$ LANGUAGE PLPGSQL STRICT SECURITY DEFINER;
 COMMENT ON FUNCTION vibetype_test.account_update_address_coordinates(UUID, DOUBLE PRECISION, DOUBLE PRECISION) IS 'Updates an account''s location based on latitude and longitude (GPS coordinates).';
 
 GRANT EXECUTE ON FUNCTION vibetype_test.account_update_address_coordinates(UUID, DOUBLE PRECISION, DOUBLE PRECISION) TO vibetype_account;
+
+
+CREATE OR REPLACE FUNCTION vibetype_test.account_check(
```

Comment:
All functions that test something have the suffix `_test` except those newly introduced here, I'll unify the naming.

---

When we have a higher test coverage in the future, it will likely be relevant to be more precise about what a function tests (like a SELECT policy) when naming a test function, but not for now.

Reply:
Changed, you can resolve when read